binaries/scheduler/main.go

```diff
@@ -17,21 +19,22 @@ func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
 	flag.Parse()
 
-	parser := config.DefaultParser()
-	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
-	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
-	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	env, o := config.Default()
```

Comment:
Cool, that sounds great to me. I know Jason had some concerns about how to make it easy to iterate on development, so it sounds like you two have the most thoughts about Config.

My focus is that Open Source Scoot is easy to use as a library and compose into a binary with different implementations of interfaces (and without having to repeat a lot of boilerplate)